README.mediawiki

```diff
@@ -1230,6 +1230,7 @@ Installing this ZenPack will add the following items to your Zenoss system.
 
 ;2.6.11
 * Fix Monitoring of SQL Instances results in a UUID error (ZPS-453)
+* Added additional event classes and handlers for Kerberos failures (ZEN-25700)
```

Comment:
It's third entry in the Changes section regarding the ticket, should I remove the previous two entries?